whill_driver/src/whill_node.cpp

```diff
@@ -227,6 +229,30 @@ void WhillNode::OnSetBatteryVoltageOutSrv(
   RCLCPP_WARN(this->get_logger(), "BatteryVoltageOut command is not available on Model CR2!!");
 }
 
+void WhillNode::OnSetBatterySavingSrv(
+  const std::shared_ptr<rmw_request_id_t> request_header,
+  const std::shared_ptr<whill_msgs::srv::SetBatterySaving::Request> request,
+  const std::shared_ptr<whill_msgs::srv::SetBatterySaving::Response> response)
+{
+  (void)request_header;
+  response->result = -1;
+
+  uint8_t l0 = uint8_t(request->l0);
+  uint8_t b0 = uint8_t(request->b0);
+  if (IsOutside(l0, 1, 90)) {
+    RCLCPP_WARN(this->get_logger(), "l0 must be assingned between 1 - 90");
+    return;
+  }
+  if (IsOutside(b0, 0, 1)) {
+    RCLCPP_WARN(this->get_logger(), "b0 must be assingned between 0 - 1");
+    return;
+  }
+
+  RCLCPP_INFO(this->get_logger(), "Battery saving settings are set");
```

Comment:
```suggestion
  RCLCPP_INFO(this->get_logger(), "Battery saving settings are set (l0: %d, b0: %d)", l0, b0);
```

ここはログに設定値を出しておいたほうがあとから見たときに便利かもしれません